src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -58,7 +58,7 @@
     <div class="col primary">@await DisplayAsync(Model.Content)</div>
 }
 
-<script at="Foot" type="text/javascript">
+<script asp-name="removeEmptyActionsMenu" at="Foot" type="text/javascript">
     $(function(){
         
         $("#actionsMenu.dropdown-menu").each(function(i, e){
```

Comment:
Looks good. 

Do you mind changing the script where it uses an id of `actionsMenu` to use a class of `.actions-menu` as well?

I doubt it'll work properly rendering multiple id's in the dom, and is never good practise

Reply:
You're absolutely right, Dean.

I added a second commit.